src/main/java/com/contrastsecurity/maven/plugin/AbstractContrastMavenPluginMojo.java

```diff
@@ -52,6 +54,7 @@
   @Parameter(property = "minSeverity", defaultValue = "Medium")
   protected String minSeverity;
 
+  // TODO[JG] why is this required?
```

Comment:
I think I see what's happening.

The server name is not required, because the agent will derive one (e.g. the hostname). The user may override the agent's default server name with this configuration. The "verify" mojo attempts to find vulnerabilities that were discovered earlier in the Maven run (during testing, after the "install" mojo ran). This plugin was built before Contrast had a good way to isolate vulnerabilities using sessions (the build-based views feature), so the plugin attempts to filter the vulnerabilities with tags and server IDs. If the plugin forces the user to set the server name to something relatively unique in their environment like "ci-server", then the "verify" filter criteria is more likely to be able to distinguish vulnerabilities found in this environment (vs if the agent used a crappy default name like "localhost").

I think we'll take care of this when we change the plugin to use sessions instead of trying to filter vulnerabilities the way it is doing here